exporter/alertmanagerexporter/alertmanager_exporter_test.go

```diff
@@ -293,9 +293,7 @@ func newMockServer(t *testing.T) *MockServer {
 		_, errWrite := fmt.Fprint(w, "test")
 		assert.NoError(t, errWrite)
 		_, err := io.ReadAll(r.Body)
-		if err != nil {
-			t.Fatal(err)
-		}
+		assert.NoError(t, err)
```

Comment:
Should these be changed to `require.NoError(t, err)` to stop the test immediately?